packs/git/config.yaml

```diff
@@ -1,2 +1,6 @@
 ---
   url: "git@demo-git:/home/git/repos/test.git"
+
+  # For Web-Based Requests
+  user: ""
```

Comment:
Hm, those two settings seems to be a leftover from a github pack and are not really used here or?